cypress/integration/pages/homePage/tests.js

```diff
@@ -1,8 +1,25 @@
+import HOME_PAGE_CONFIG from '../../../../src/app/routes/homePage/getInitialData/page-config';
+
 export default ({ service, pageType }) => {
-  it.skip(`should render a simple ${service} ${pageType} page`, () => {
-    cy.get('[data-testid="home-page"]').should(
-      'contain',
-      'Hi, I am a Home Page component',
-    );
+  describe(`Running tests for ${service} ${pageType}`, () => {
+    let articlesData;
+    before(() => {
+      const env = Cypress.env('APP_ENV');
+      cy.getPageData({
+        service,
+        pageType: 'home',
+        id: HOME_PAGE_CONFIG[service][env],
+      }).then(({ body }) => {
+        articlesData = body;
+      });
+    });
+
+    it.skip(`should render a simple ${service} ${pageType} page`, () => {
+      cy.log(articlesData);
```

Comment:
It won't run this code unless we unskip the test. But also, let's remove it before we merge.

```suggestion
```

Reply:
We can't unskip the test yet, because it will fail 😿 This is where we can eventually add the new test for the Chartbeat Config though 😸